src/assets/main.css

```diff
@@ -2,6 +2,31 @@
 
 html, body {
     font-family: 'Roboto', sans-serif;
+    margin: 0 auto;
+
+}
+body {
+    margin: 0 0 0 62px;
+    background-color: #f4f6fc;
+    color: rgb(40, 54, 75);
+}
+
+div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
```

Comment:
How are universal typography and colors defined in the company codebase?
Are they usually in main.css, how is LESS incorporated in it?

Reply:
see src/styles/* in aksnicelydo